src/metrics/metrics.md

```diff
@@ -1,11 +1,61 @@
 # Metrics
 
-Here we provide definitions and information about our standard metrics.
+This section contains definitions and information about standard metrics used at Mozilla.  You may wish to refer to the [terminology section](../concepts/terminology.html) while reading this document, in case a particular concept is not clear.
 
-Table of contents with links to all metrics.
+## Table of Contents
+
+<!-- toc -->
 
 -----
 
+## Daily Active Users (DAU)
+
+### Overview
+
+Daily Active Users or DAU counts the number of unique profiles active in the product on each day.  This is intended to approximate the number of people using the product each day.
+
+### Details
+
+DAU counts unique profiles. Keep in mind that a [profile is not necessarily a user](/concepts/analysis_gotchas.html#profiles-vs-users).
+
+The standard concept of active varies by product, but generally, active users are defined as unique profiles that have sent a `main` ping (on desktop) or a `baseline` ping (on mobile).  The precise criteria are defined in the [`usage criterion`](./usage.md) section of this documentation.
+
+We can also restrict the metric to alternative usage critera. It's **critical to clearly state any non-standard usage criteria on the metric**. The metrics team suggest the following format: `DAU(usage criterion)`. For example, we might be interested in the count of profiles that have viewed more than 5 URIs in a day. We'd denote that metric as `DAU(URI > 5)`.
+
+Some common alternative usage criteria documented in the [`usage criterion` section](./usage.md).
+
+### Caveats
+
+If the number of users stays constant, but the average number of active profiles per user increases, this metric will tend to increase.  For more details on the relationship between users and profiles, see [the profiles vs users section in analysis gotchas](https://docs.telemetry.mozilla.org/concepts/analysis_gotchas.html#profiles-vs-users).
+
+### Dashboards
+
+This metric is available on the [standard Growth and Usage Dashboard (GUD)](https://go.corp.mozilla.com/gud) for most products and with some slicing available.
+
+### Tables
+
+DAU can easily be calculated from the [Exact MAU tables](https://docs.telemetry.mozilla.org/datasets/bigquery/exact_mau/reference.html); for example:
+
+```sql
```

Comment:
Similar to [this comment thread on the day 2-7 activation doc](https://docs.google.com/document/d/1HwlX6E3NzPkudVLBJXPcS49XpgwfxIIugXO0Y7kNlKM/edit?disco=AAAAGrjNdTQ), it would be good if there was a query that used a less-raw dataset - one that is more useful to data scientists wanting to do new things.

But I don't see this as a blocker for this PR, because I think DSes are going to either know how to do it, work out how to do it, or can just ask for help in less time than it would take to edit this document

Reply:
Totally agree.  We discussed that above and I think having an example from both the highest-level and easiest table and also the rawest.  Will defer that for the next PR though.